core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -42,7 +42,7 @@
       <div class="form-group row">
         <label for="label-target-user-bio" class="col-lg-4 col-md-4 col-sm-4" translate="I18N_PREFERENCES_BIO"></label>
         <div class="col-lg-8 col-md-8 col-sm-8">
-          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-blur="$ctrl.saveUserBio($ctrl.userBio)" rows="5" maxlength="2000"></textarea>
+          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-blur="$ctrl.saveUserBio($ctrl.userBio)" ng-change="$ctrl.checkBioChanged()" rows="5" maxlength="2000"></textarea>
```

Comment:
optional: can we use ng-focus instead of ng-change? (not sure!) [Asking this just to avoid duplicate function calls while the user is editing the bio.]

Reply:
No, If the textbox is focused and there is no change in the text then the save function won't be called to update the allowreload value to false